frontend/src/fixtures/userCommonsFixtures.js

```diff
@@ -1,4 +1,14 @@
 const userCommonsFixtures = {
+    // Backend does not include "commons" or "user", so they probably shouldn't be in these fixtures
+    oneRealUserCommons: {
+        "id": 1,
+        "userId": 1,
+        "commonsId": 1,
+        "username": "idk if this field is even used",
+        "totalWealth": 1000,
+        "numOfCows": 5,
+        "cowHealth": 98.0
+    },
```

Comment:
Not sure if this is needed since it's not being used

Reply:
It's not needed in this PR, but I use it in later ones. I included it because the inaccurate fixtures caused me a ton of issues since I thought I could access commons through userCommons, which doesn't work.